sdk/internal/recording/recording_test.go

```diff
@@ -550,6 +551,63 @@ func TestHostAndScheme(t *testing.T) {
 	require.Equal(t, r.host(), "localhost:5000")
 }
 
+func TestGitRoot(t *testing.T) {
+	cwd, err := os.Getwd()
+	require.NoError(t, err)
+	_, err = getGitRoot(cwd)
+	require.NoError(t, err)
+}
+
+func TestGitRootNotFound(t *testing.T) {
+	_, err := getGitRoot("../../../../")
```

Comment:
I'm not a fan of relative pathing.  Could this be a variable we pass in or something set on the environment?

Reply:
I'm going to switch to shelling out to git to get the root as Joel suggested below, so I'll be able to remove this test.